make/source.mk

```diff
@@ -245,6 +246,7 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             rx/sbus.c \
             rx/sbus_channels.c \
             rx/spektrum.c \
+            io/spektrum_vtx_control.c \
```

Comment:
Does this need to be speed optimised? None of the other VTX control functions is.

Reply:
I assumed so since it is still in the scheduler inner loop. Once/if that is fixed, it should not be need any speed opt. Also, originally when it was part of spektrum.c it was speed optimized, and to make no functional or performance changes it should still be so. 
And you can not compare to other VTX control functions, this is VTX control INPUT in the RC-Rx end. The other are control OUTPUT towards the VTXs. (Mainly, not sure about all. Are there any other VTX ctrl input there?).

Reply:
I think we should fix this now, and not wait (and then forget about it). Remember that this is not a version to be released in the near future, so there is no reason to not want to make functional or performance changes. It makes even less sense to me to optimise this, but not optimise the second half of the VTX functionality that this is calling into.

Reply:
OK, you got a point there. Called output VTX functions not beeing speed optimized. Will fix.